src/Uno.UI/UI/Xaml/Media/EllipseGeometry.wasm.cs

```diff
@@ -41,6 +44,31 @@ private void OnPropertyChanged(ManagedWeakReference instance, DependencyProperty
 			}
 		}
 
-		internal override SvgElement GetSvgElement() => _svgElement;
+		internal override SvgElement GetSvgElement()
+		{
+			if (_svgElement == null)
+			{
+				_svgElement = new SvgElement("ellipse");
+#if DEBUG
+				_svgElement.SetAttribute("uno-geometry-type", "EllipseGeometry");
+#endif
+
+				OnPropertyChanged(null, CenterProperty, null);
```

Comment:
Feels weird notifying PropertyChanged from a Get Method.

Reply:
It's not generating the event... it's just initializing the properties from their current value.